bin/live/pycbc_live_single_significance_fits

```diff
@@ -424,7 +326,8 @@ with h5py.File(args.output, 'w') as fout:
     fout['bins_lower'] = tbins.lower()
 
     fout.attrs['ifos'] = ','.join(args.ifos)
-    fout.attrs['analysis_date'] = args.analysis_date
+    fout.attrs['fit_start'] = args.gps_start_time
+    fout.attrs['fit_end'] = args.gps_end_time
```

Comment:
See my comment above.

Reply:
Might want to be even more explicit in fact, and use `fit_start_gps_time` / `fit_end_gps_time`.